src/main/java/chess/controller/ChessViewController.java

```diff
@@ -0,0 +1,15 @@
+package chess.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/chess-game")
+public class ChessViewController {
+
+    @GetMapping()
```

Comment:
```suggestion
    @GetMapping
```

값이 없을 땐 괄호도 생략 가능해요.